python/kserve/test/test_inference_service_client.py

```diff
@@ -22,7 +22,7 @@
 from kserve import V1beta1InferenceService
 from kserve import KServeClient
 
-kserve_client = KServeClient(config_file='./kserve/test/kubeconfig')
+kserve_client = KServeClient(config_file='./test/kubeconfig')
```

Comment:
q about this - if you run from the Makefile, you need the path I changed it to. The Makefile command wont work as is with this path, so it is broken. Should we change it?

Reply:
looks like his is breaking the test, probably better to change the Makefile